guides/doc-Release_Notes/topics/katello-4.10.0.adoc

```diff
@@ -0,0 +1,152 @@
+= Katello 4.10.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1687[Redmine]
+
+== Katello
+
+* Incorrect aria-label and component id in the alternate content source modal for editing credentials certificates - https://projects.theforeman.org/issues/36512[#36512]
```

Comment:
```suggestion
* Incorrect aria-label and component ID in the alternate content source modal for editing credentials certificates - https://projects.theforeman.org/issues/36512[#36512]
```
Even below.

Reply:
Thanks! Pushed these edits and other typos where / was present for some characters